build.gradle

```diff
@@ -23,17 +23,22 @@ plugins {
     id 'java-library'
     id 'org.bytedeco.gradle-javacpp-platform' version '1.5.9'
     id 'io.freefair.lombok' version '8.4'
+    id 'maven-publish'
```

Comment:
Please remove this plugin from this pull request. This PR is named "update gradle" and should not be mixed with other stuff. You are very welcome to add another PR with the Maven stuff!

Reply:
fixed